config.json

```
@@ -2,9 +2,6 @@
     "name": "Image Map",
     "namespace": "ImageMap\\ExternalModule",
     "description": "This module replaces an input, radio, or checkbox field with an interactive image where users can click on image regions to select one or more field options. Find the complete <b><a href='https://github.com/ctsit/imagemap' target='_blank'>documentation</a></b> and an example instrument at <a href='https://github.com/ctsit/imagemap/blob/master/docs/Instrument%20Example.zip' target='_blank'>https://github.com/ctsit/imagemap</a>.",
-    "permissions": [
-        "redcap_every_page_top"
-    ],
     "framework-version": 9,
```

```suggestion
    "framework-version": 14,
```
Bump framework version to 14 to accommodate removal of permissions section

---

GitHub, in their infinite wisdom, have removed my ability to commit this directly.